orttraining/orttraining/training_api/checkpoint.cc

```diff
@@ -171,10 +176,9 @@ Status ToFile(const PathString& checkpoint_path, flatbuffers::FlatBufferBuilder&
  * @param checkpoint_path file where checkpoint is saved.
  * @return Status
  */
-Status FromTensorProtos(
-    gsl::span<const ONNX_NAMESPACE::TensorProto> trainable_tensor_protos,
-    gsl::span<const ONNX_NAMESPACE::TensorProto> non_trainable_tensor_protos,
-    const PathString& checkpoint_path, const bool nominal_checkpoint) {
+Status FromTensorProtos(gsl::span<const ONNX_NAMESPACE::TensorProto> trainable_tensor_protos,
```

Comment:
Should we also consider updating the function `save::FromCheckpointState` for saving the checkpoint to an external data on the device?